easybuild/easyconfigs/o/OpenMPI/OpenMPI-1.8.6-GNU-5.1.0-2.25.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '1.8.6'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'GNU', 'version': '5.1.0-2.25'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+
+dependencies = [('hwloc', '1.11.0')]
+
+configopts = '--with-threads=posix --enable-shared --enable-mpi-thread-multiple --with-verbs '
+configopts += '--enable-mpirun-prefix-by-default '  # suppress failure modes in relation to mpirun path
+configopts += '--with-hwloc=$EBROOTHWLOC '  # hwloc support
+configopts += '--disable-dlopen '  # statically link component, don't do dynamic loading
+
+# needed for --with-verbs
+osdependencies = [('libibverbs-dev', 'libibverbs-devel')]
+
+sanity_check_paths = {
+    'files': ["bin/%s" % binfile for binfile in ["ompi_info", "opal_wrapper", "orterun"]] +
+             ["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in ["mpi_cxx", "mpi_mpifh",
+                                                                         "mpi", "ompitrace", "open-pal",
+                                                                         "open-rte", "vt", "vt-hyb",
+                                                                         "vt-mpi", "vt-mpi-unify"]] +
```

Comment:
move all of these to the left please, letting them hang all at the right there might make them invisible on smaller displays.
```
["lib/lib%s.%s" % (libfile, SHLIB_EXT) for libfile in [
    "mpi_cxx", "mpi_mpifh", "mpi", "ompitrace", "open-pal",
    "open-rte", "vt", "vt-hyb", "vt-mpi", "vt-mpi-unify",
    ]
 ]
```